docs/HomeGettingStarted.md

```diff
@@ -11,7 +11,7 @@ If you are using a module bundler, such [Webpack](https://webpack.js.org/), [Bro
 1. Add TwitchJS to your project:
    ```bash
    npm install --save twitch-js
-   ```

```

Comment:
We still need this.

(sorry for backseat driving)

Reply:
Oops, my bad

Reply:
No worries. We are all learning.